app/models/mixins/cloud_tenant_mixin.rb

```diff
@@ -1,8 +1,18 @@
 module CloudTenantMixin
   extend ActiveSupport::Concern
 
-  # TODO(lpichler) add synchronization
+  OPENSTACK_CLOUD_PROVIDER = 'Openstack Cloud Provider'.freeze
```

Comment:
Hrm, not crazy about exposing `openstack` in a base mixin.  I'll read through the rest to see if a better idea comes to mind.

/cc @durandom